### DISCUSSION TITLE
Accessing Windows networking apps from WSL2

### DISCUSSION THREAD

**Discussion by tfl-takashima** [sakai135/wsl-vpnkit#52] [Q&A] [answered]
Hi Team,

https://docs.microsoft.com/en-us/windows/wsl/networking#accessing-windows-networking-apps-from-linux-host-ip

My WSL2 Environment with wsl-vpnkit enabled can not access to windows apps as mentioned above and it is possible only when I specify the ip address of VPN adapter as the URL. Specifying the nameserver ip address does not work.

Could you check if you can reproduce the issue above on your side?

Regards,

**Comment by sakai135** [comment, marked as answer]
This is currently expected behavior when using wsl-vpnkit. As noted in the readme [here](https://github.com/sakai135/wsl-vpnkit#notes), you can access the Windows networking apps using `host.internal` or `192.168.67.2`.

This is a holdover from earlier versions of the script, and I should be able to recreate the behavior as the MS WSL documentation describes.

**Reply by tfl-takashima** [reply]
Hi @sakai135 

I'm terribly sorry for asking without checking docs.


Regards,

**Reply by sakai135** [reply]
Don't be sorry. I would rather not deviate from MS docs when possible, so your question was a great prompt. [v0.2.2](https://github.com/sakai135/wsl-vpnkit/releases/tag/v0.2.2) fixes this.
